### PR TITLE
(google) Implement GoogleImageFinder so findImageFromTags works for G…

### PR DESCRIPTION
**Pull request by duftler**
…CE provider.

### CODE REVIEW

**Comment by duftler** [comment]
Implementation is quite close to the `AmazonImageFinder`, but image details are just different enough to warrant keeping the logic separate.

**Comment by duftler** [comment]
@danielpeach please review.

**Comment by duftler** [comment]
Related to: https://github.com/spinnaker/clouddriver/pull/1347.

File: orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleImageFinderSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.tasks.providers.gce
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.orca.clouddriver.OortService
+import com.netflix.spinnaker.orca.pipeline.model.Pipeline
+import com.netflix.spinnaker.orca.pipeline.model.PipelineStage
```
Comment (danielpeach) [inline orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleImageFinderSpec.groovy]: I don't think these two imports are needed.

Comment (duftler) [inline orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/gce/GoogleImageFinderSpec.groovy]: Done.

**Pull request merged**